conductor-config.yaml

```diff
@@ -0,0 +1,6 @@
+network:
+  network_type: quic_bootstrap
+  transport_pool:
+    - type: webrtc
+      signal_url: "wss://signal.holo.host"
+  bootstrap_service: "https://bootstrap.holo.host"
```

Comment:
We should probably write a story to move away from production services for performance testing.

Reply:
Yes, absolutely - https://github.com/holochain/wind-tunnel/issues/52